src/shared/model/QueryParams.ts

```diff
@@ -28,6 +28,7 @@ export interface QueryParams
   extends PersistableQueryParams,
     StringifiableRecord {
   emailVerified?: boolean;
+  emailSentPage?: boolean;
```

Comment:
This query parameter is used to determine if the request from sent from an email sent page, for example when resending an email. We use this parameter to determine whether to show or hide the "Email sent" success message that appears when you resend an email.

Reply:
ah, I left a comment above before seeing your comment here. I think I wasn't sure what the meaning was until I read your comment/how it was used. Maybe a name like `fromEmailSentPage` or `emailSentSuccess` could be more self explanatory 🤔

Reply:
I like `emailSentSuccess`! I'll change it to that.